packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -654,4 +710,80 @@ export class PluginLedgerConnectorQuorum
     }
     return this.runDeploy(req);
   }
+
+  // Low level function to call any method from web3.eth
+  // Should be used only if given functionality is not already covered by another endpoint.
+  public async invokeRawWeb3EthMethod(
+    args: InvokeRawWeb3EthMethodV1Request,
+  ): Promise<any> {
+    return new Promise((resolve, rejects) => {
+      this.log.debug("invokeRawWeb3EthMethod input:", JSON.stringify(args));
+
+      Checks.nonBlankString(
+        args.methodName,
+        "web3.eth method string must not be empty",
+      );
+
+      const looseWeb3Eth = this.web3.eth as any;
+      if (
+        Object.prototype.hasOwnProperty.call(looseWeb3Eth, args.methodName) &&
+        typeof looseWeb3Eth[args.methodName] === "function"
+      ) {
+        const web3Method = looseWeb3Eth[args.methodName];
+
+        const web3Response: Promise<unknown> = args.params
+          ? web3Method(...args.params)
+          : web3Method();
+
+        web3Response
```

Comment:
@outSH Could this be made easier to read with an `await` and a try/catch?

Reply:
@petermetz Yeah, while writing the function I was trying to "trick" CodeQL into  seeing that this execution path runs only when method is verified, but it didn't help :( 

I've refactored this function into smaller and less convoluted one.

Reply:
> @petermetz Yeah, while writing the function I was trying to "trick" CodeQL into seeing that this execution path runs only when method is verified, but it didn't help :(
> 
> I've refactored this function into smaller and less convoluted one.

@outSH Great, thank you so much!